astra-sim/system/Sys.hh

```diff
@@ -309,6 +309,9 @@ class Sys : public Callable {
 
   // statistics
   bool trace_enabled;
+
+  // replay operators based on their runtime and do not run performance models
```

Comment:
@beckmabd Updated the comment.

Reply:
Better to say "skip simulation for operator and use current duration". 

Replay means other things given we have chakra/param replay?

Reply:
@srinivas212 , Updated, thanks!